components/ContentEditor/ContentEditor.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'React'
+import { Editor, EditorState, convertFromRaw, RawDraftContentState, convertToRaw } from 'draft-js'
+import 'draft-js/dist/Draft.css'
+
+type ContentEditorType = {
+  onSave: (content: object) => void
+}
+
+const ContentEditor = ({ onSave }: ContentEditorType) => {
+  const [editorState, setEditorState] = useState(EditorState.createWithContent(emptyContentState))
+
+  return (
+    <div>
+      <Editor
+        editorKey={'editorKey'}
+        editorState={editorState}
+        onChange={(data) => {
+          console.log('data: ', data)
+          const content = editorState.getCurrentContent();
+          console.log('content: ', convertToRaw(content));
```

Comment:
Which is the best way to store draft js editor content?

<img width="905" alt="Screen Shot 2022-10-17 at 18 00 27" src="https://user-images.githubusercontent.com/223620/196212076-20f8c489-f171-4cd4-a273-3f11af04a3ec.png">